src/index.html

```diff
@@ -1,13 +1,62 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE html>
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div class="card" data-qa="card">
+      <img 
+        src="/images/imac.jpeg" 
+        alt="imac"
+        class="card__image"
+      >
+
+      <div class="card__content">
+        <div class="card__name">
+          <h1 class="card__h1">
```

Comment:
```suggestion
          <h1 class="card__title">
```
Avoid using tag names as block element names, name should be more descriptive